sandbox/vertical-video.html.example

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <title>Video.js Sandbox</title>
+  <link href="../dist/video-js.css" rel="stylesheet" type="text/css">
+  <script src="../dist/video.js"></script>
+</head>
+<body>
+  <div style="background-color:#eee; border: 1px solid #777; padding: 10px; margin-bottom: 20px; font-size: .8em; line-height: 1.5em; font-family: Verdana, sans-serif;">
+    <p>You can use /sandbox/ for writing and testing your own code. Nothing in /sandbox/ will get checked into the repo, except files that end in .example (so don't edit or add those files). To get started run `npm start` and open the vertical-volume.html</p>
+    <pre>npm start</pre>
+    <pre>open http://localhost:9999/sandbox/vertical-video.html</pre>
+  </div>
+
+  <!--  To make this truly responsive, we have to constrain the container's width.        -->
+  <!--  Because vertical videos are concerned more with height, we base it off vh units.  -->
+  <!-- calc(100vh * 0.5625) is equivalant to "screen height * (9:16)"                     -->
+ <div style="
+        width: calc(100vh * 0.5625); 
+        max-width: 100%;
+        margin: 0 auto;">
```

Comment:
I think this outer div can be removed and instead the aspect ratio option be passed to the player.

Reply:
Does the fluid nature of VideoJS take the screen height into account? That was not my experience with it.

Reply:
It's possible that a `min-height` or something like that should be added to the player div. But with the aspect ratio it should mostly do the right thing.